using-mobile-extensions/adobe-edge-consent/README.md

```diff
@@ -0,0 +1,165 @@
+# Adobe Experience Platform Edge Consent
+
+## Configure the Adobe Experience Platform Edge Consent extension in Experience Platform Launch
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Consent Collection** extension, and click **Install**.
+3. Set your desired default consent level.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+### Configure the AEP Edge Consent extension
+
+![Adobe Edge Network extension configuration](/Users/nporter/Developer/Documentation/acp-sdks-documentation/.gitbook/assets/mobile-edge-consent-launch-configuration.png)
+
+
+
+## Add the AEP Edge Consent extension to your app
+
+### Download and import the Edge Consent extension
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Java
+
+1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
+
+   ```java
+   implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
```

Comment:
`implementation 'com.adobe.marketing.mobile:core:1.+'` Core instead of SDK-Core ?
You'll need to individually include Lifecycle and Signal as well.